xbmc/Util.cpp

```diff
@@ -2114,20 +2114,8 @@ ExternalStreamInfo CUtil::GetExternalStreamDetailsFromFilename(const std::string
     std::vector<std::string> tokens;
     StringUtils::Tokenize(inputString, tokens, delimiters);
 
-    for (std::vector<std::string>::iterator it = tokens.begin(); it != tokens.end(); ++it)
+    for (std::vector<std::string>::reverse_iterator it = tokens.rbegin(); it != tokens.rend(); ++it)
```

Comment:
`std::vector<std::string>::reverse_iterator` iterator can be changed with `auto`

Reply:
done - i think... :)